diff_drive_controller/package.xml

```diff
@@ -14,12 +14,16 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
+  <build_depend>cmake_modules</build_depend>
   <build_depend>controller_interface</build_depend>
   <build_depend>nav_msgs</build_depend>
   <build_depend>realtime_tools</build_depend>
   <build_depend>tf</build_depend>
   <build_depend>urdf</build_depend>
+  <build_depend>eigen</build_depend>
```

Comment:
I believe so. `cmake_modules` only gets you the ability to `find_package` it, but you still need the package itself to be present on the system.